### PR TITLE
[1LP][RFR] Converting clusters to Widgetastic

### PR DESCRIPTION
**Pull request by rsnyman**
Converting Computer/Infrastructure/Clusters to Widgetastic

### CODE REVIEW

File: cfme/infrastructure/cluster.py [hunk 1]
```diff
@@ -65,45 +157,83 @@ def __init__(self, name, provider, appliance=None):
...
+            table = view.totals_for_hosts
+        elif ident[0] == 'Totals for VMs':
+            table = view.totals_for_vms
+        elif ident[0] == 'Configuration':
+            table = view.configuration
+        elif ident[0] == 'Smart Management':
+            table = view.smart_management
+        if table:
+            return table.get_text_of(ident[1])
+        return None
```
Comment (quarckster) [inline cfme/infrastructure/cluster.py]:
```python
getattr(view, ident[0].lower().replace(" ", "_")).get_text_of([ident[1]])
```

Comment (quarckster) [inline cfme/infrastructure/cluster.py]: @rsnyman there was a typo the correct line is:
```python
getattr(view, ident[0].lower().replace(" ", "_")).get_text_of(ident[1])
```

File: cfme/infrastructure/cluster.py [hunk 2]
```diff
@@ -4,48 +4,140 @@
...
     """ Model of an infrastructure cluster in cfme
 
-    Args:
-        name: Name of the cluster.
-        provider: provider this cluster is attached to.
+    :param name: Name of the cluster.
+    :param provider: provider this cluster is attached to.
 
-    Note:
+    .. _note:
```
Comment (psav) [inline cfme/infrastructure/cluster.py]: Whilst there may be preference here, please stick to the current way of expressing docstrings to maintain consistency

File: cfme/tests/infrastructure/test_cluster_analysis.py [hunk 3]
```diff
@@ -22,8 +22,9 @@ def test_run_cluster_analysis(request, setup_provider, provider, soft_assert):
         test_flag: cluster_analysis
     """
     cluster_name = provider.data['remove_test']['cluster']
-    test_cluster = cluster.Cluster(name=cluster_name, provider=provider)
-    wait_for(lambda: test_cluster.exists, delay=10, num_sec=120, fail_func=sel.refresh)
+    test_cluster = Cluster(name=cluster_name, provider=provider)
+    # wait_for(lambda: test_cluster.exists, delay=10, num_sec=120, fail_func=sel.refresh)
```
Comment (psav) [inline cfme/tests/infrastructure/test_cluster_analysis.py]: Why keeping the comment?

**Pull request merged**